phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataProtocol.java

```diff
@@ -118,8 +118,9 @@ public abstract class MetaDataProtocol extends MetaDataService {
     /** Version below which we fall back on the generic KeyValueBuilder */
     public static final int CLIENT_KEY_VALUE_BUILDER_THRESHOLD = VersionUtil.encodeVersion("0", "94", "14");
     // Version at which we allow SYSTEM.CATALOG to split
-    public static final int MIN_SPLITTABLE_SYSTEM_CATALOG = VersionUtil.encodeVersion("5", "1", "0");
-    public static final String MIN_SPLITTABLE_SYSTEM_CATALOG_VERSION = "5.1.0";
+    public static final int MIN_SPLITTABLE_SYSTEM_CATALOG =
```

Comment:
Cc @jpisaac

Reply:
@gjacoby126  Did we even release 5.0 ? I don't see it in tags here: https://github.com/apache/phoenix/tags

Reply:
If the server is non-splitable (v5.0.0), then the client (v5.0.0) would also be not split aware right? Are we envisioning scenarios like 4.16 clients and v5.0.0 servers?

Reply:
As of today, 4.16 client would not be able to connect to 5.0 server as [this commit](https://github.com/apache/phoenix/commit/e516b19e8a6c0f5d2eea4bc7cd6ecbe611bf300e) only allows it to connect to 5.1 server.

Reply:
The other possibility is 5.0.0 client and 5.1.x server.

Reply:
@jpisaac - 5.0 client and 5.1 server was the combination I was worried about. That should be allowed (it's necessary, for example, if you're running 5.0 and need to upgrade to 5.1 during the transition)

Reply:
@shahrs87 - Phoenix 5.0 was an official release -- it's the HBase 2.x equivalent of 4.14. https://blogs.apache.org/phoenix/entry/apache-phoenix-releases-next-major

Reply:
This complicates the checks for MIN_SPLITTABLE_SYSTEM_CATALOG (e.g => clientVersion < MIN_SPLITTABLE_SYSTEM_CATALOG)
Since  MIN_SPLITTABLE_SYSTEM_CATALOG  is not fixed it will need to be client major version dependent.
@gjacoby126 @shahrs87 @virajjasani